test/boost/cdc_test.cc

```diff
@@ -194,69 +204,82 @@ SEASTAR_THREAD_TEST_CASE(test_with_cdc_parameter) {
     }).get();
 }
 
-SEASTAR_THREAD_TEST_CASE(test_detecting_conflict_of_cdc_log_table_with_existing_table) {
+SEASTAR_THREAD_TEST_CASE(test_conflict_with_existing_table_when_cdc_enabled_via_alter_table) {
     do_with_cql_env_thread([] (cql_test_env& e) {
         // Conflict on CREATE which enables cdc log
         e.execute_cql("CREATE TABLE ks.tbl_scylla_cdc_log (a int PRIMARY KEY)").get();
         BOOST_REQUIRE_THROW(e.execute_cql("CREATE TABLE ks.tbl (a int PRIMARY KEY) WITH cdc = {'enabled': true}").get(), exceptions::invalid_request_exception);
         BOOST_REQUIRE(!e.local_db().has_schema("ks", "tbl"));
 
-        e.execute_cql("CREATE TABLE ks.tbl (a int PRIMARY KEY, b vector<float, 3>)").get();
+        e.execute_cql("CREATE TABLE ks.tbl (a int PRIMARY KEY)").get();
         BOOST_REQUIRE(e.local_db().has_schema("ks", "tbl"));
 
         // Conflict on ALTER which enables cdc log
         BOOST_REQUIRE_THROW(e.execute_cql("ALTER TABLE ks.tbl WITH cdc = {'enabled': true}").get(), exceptions::invalid_request_exception);
+    }).get();
+}
+
+SEASTAR_THREAD_TEST_CASE(test_conflict_with_existing_table_when_cdc_enabled_via_vector_index) {
+    do_with_cql_env_thread([] (cql_test_env& e) {
+        // Conflict on CREATE which enables cdc log
+        e.execute_cql("CREATE TABLE ks.tbl_scylla_cdc_log (a int PRIMARY KEY)").get();
+        BOOST_REQUIRE_THROW(e.execute_cql("CREATE TABLE ks.tbl (a int PRIMARY KEY) WITH cdc = {'enabled': true}").get(), exceptions::invalid_request_exception);
```

Comment:
This command (and the next one) doesn't belong in this test, it was already tested in the previous test, and this test is supposed to be about vector index, not CREATE TABLE.